frontend/src/model/beta/reviews/ReviewDisplay.tsx

```diff
@@ -63,10 +70,22 @@ export default function ReviewDisplay({ review }: ReviewDisplayProps) {
   return (
     <>
       {isModelRolesLoading && <Loading />}
-      <Stack>
-        {acceptedReviewResponses}
-        {changesRequestedReviewResponses}
-      </Stack>
+      {changesRequestedReviewResponses.length === 0 && acceptedReviewResponses.length > 0 && approvalsDisplay()}
+      {changesRequestedReviewResponses.length > 0 && (
+        <Tooltip
+          title={`${changesRequestedReviewResponses.length} ${
+            changesRequestedReviewResponses.length > 1 ? 'reviews' : 'review'
```

Comment:
This can be simplified by using the `plural` util function in `frontend/utils/stringUtils.ts`